supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -0,0 +1,30 @@
+[[support]]
+= Support
+
+[[technology-preview-guidance]]
+== Technology Preview
+
+Technology Preview features provide early access to upcoming product innovations, enabling customers to test functionality and provide feedback during the development process. However, these features are not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.
+
+When documenting a Technology Preview feature, follow these guidelines:
+
+* Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content.
+* Use initial uppercase capitalization, that is, Technology Preview.
+* Include a brief description of the Technology Preview feature in the release notes.
+* Maintain a list of features that are currently in Technology Preview status in the release notes.
+* Never use "supported as a Technology Preview", and avoid using "support" in Technology Preview descriptions. Instead, use neutral words like "available", "provide", "capability", "functionality", "implement", "enable", and so on.
```

Comment:
I also agree with @bburt-rh that this list of 6 examples is too long. I'd recommend cutting it down to "available", "provide" and maybe one other one.